mix.exs

```diff
@@ -29,9 +29,7 @@ defmodule OpenTelemetry.Honeycomb.MixProject do
   end
 
   def application do
-    [
-      extra_applications: [:logger, :hackney, :poison]
-    ]
+    []
```

Comment:
Removed `logger` as well because we don't use it anywhere in the code I could see.

Reply:
I'd prefer to keep unrelated changes out.